app/frontend/src/search/client.js

```diff
@@ -0,0 +1,16 @@
+// This is the public API key which can be safely used in your frontend code.
+// This key is usable for search queries and list the indices you've got access to.
+export const search = algoliasearch('QM2YE0HRBW', '20b88d28047d5e3d60437993ad3d9c50')
+
+// TODO remove all this global scope
+export const client = indexName => instantsearch({
```

Comment:
Even outside of the global scope I'd prefer we avoided generic names like `client` please.  Maybe `algoliaClient` or `searchClient`.  Not bothered about it at this junction, but would like more specificity as we iterate.